android/Android_App/app/src/main/java/com/example/youngcarers/cards/EmergencyNumberCard.kt

```diff
@@ -0,0 +1,72 @@
+package com.example.youngcarers.cards
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.example.youngcarers.R
+import com.example.youngcarers.ui.theme.colorDarkRed
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun EmergencyNumberCard(header: String) {
+
+    Card(
+        modifier = Modifier
+            .padding(10.dp)
+            .fillMaxWidth()
+            .wrapContentHeight()
+            .clip(RoundedCornerShape(15.dp)),
+        onClick = {
+
+        },
+        shape = MaterialTheme.shapes.medium,
+        elevation = 5.dp,
+        backgroundColor = MaterialTheme.colors.surface
+
+    ) {
+
```

Comment:
Newlines in this file again.

Reply:
Please just auto-format every file in the project, as well as organize all imports.